apps/web-portal-client/lib/components/HomePage.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react'
+
+import {Logo} from './Logo'
+import {Card} from './Card'
+import {UserProfile} from './UserProfile'
+import {LoginButton} from './LoginButton'
+
+import {DesktopMenu, MobileMenu, MobileMenuButton} from './main-menu'
+
+export function HomePage() {
+  const [menuStatus, setMenuStatus] = useState(false)
```

Comment:
As for me, the "status" should contain some object or information that describes the status in a more informative way than just a boolean.
 If you need a flag that indicates a status you can use "isLogged", "setIsLogged", "showMenu", "setMenuShow"